pantheon-bundle/src/main/java/com/redhat/pantheon/model/document/Validation.java

```diff
@@ -0,0 +1,27 @@
+package com.redhat.pantheon.model.document;
+
+import com.redhat.pantheon.model.api.Field;
+import com.redhat.pantheon.model.workspace.WorkspaceChild;
+import org.apache.jackrabbit.JcrConstants;
+
+import javax.inject.Named;
+import java.util.Calendar;
+
+
```

Comment:
Should we add an annotation for @jcr:PrimaryType(pant:Validation) ?

Reply:
of course you need define pant:validation in resources/SLING_INF/nodetypes/nodetypes.cnd first